Sources/Client/Model/Channel/Channel+Requests.swift

```diff
@@ -133,6 +135,37 @@ public extension Channel {
         Client.shared.send(eventType: eventType, to: self, completion)
     }
     
+    /// Send a keystroke event for the current user.
+    /// - Parameter completion: a completion block with `Event`.
+    @discardableResult
+    func keystroke(_ completion: @escaping Client.Completion<Event>) -> Cancellable {
+        currentUserTypingTimerControl?.cancel()
+        
+        currentUserTypingTimerControl = DefaultTimer.schedule(timeInterval: 5, queue: .main) { [weak self] in
+            self?.stopTyping(completion)
+        }
+        
+        guard !isCurrentUserTyping else {
+            return Subscription.empty
+        }
```

Comment:
This feels iffy. If there are valid cases when calling `keystroke` doesn't produce a request, it needs to be reflected in the API:
- it should return `-> Cancellable?`
- `completion` should be called with a value indicating no request was actually made.

Reply:
... or at least add to the documentation comment that `completion` may never get called if `isCurrentUserTyping` is already `true`.